chef-powershell/Gemfile.lock

```diff
@@ -1,62 +1,64 @@
 PATH
   remote: .
   specs:
-    chef-powershell (1.0.10)
-      ffi (~> 1.15)
-      ffi-yajl (~> 2.4)
+    chef-powershell (18.1.0)
+      ffi (~> 1.16)
```

Comment:
We're stuck with `~> 1.15.5` until we figure out the macOS issue similar to https://github.com/ffi/ffi/issues/836